### ISSUE TITLE
Server log files do not contain timestamps

### ISSUE THREAD

**Issue by 4mfie** [OpenRA/OpenRA#16729]
I am trying to debug my dedicated servers and am in need of timestamps in the server log files in order to compare the activity from the logs with the activity shown in my server resource utilization analytics. As it stands, there is no way for me to compare events from these two data sources.

**Comment by amphyploid** [comment]
I normally run server processes detached from the console, but I am aware that timestamps are written to the console and that I could redirect the console output to a file. This may be helpful, but I've noticed that what's written to the console is incomplete. There seem to be events that are written to the log files that are not written to the console. For example, the error I experienced in #16728 appears in `server.log`, but not in the console. Either including timestamps in the log files or including timestamped errors in the console would suffice, but I'm not sure if there are technical limitations with the latter.

I'm not too familiar with this codebase, but maybe changing [OpenRA.Game/Support/Log.cs#L84](https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Support/Log.cs#L84) to:

```
writer.WriteLine("[{0}] {1}", DateTime.Now.ToFileTime(), value);
```

is all that's necessary. The the similar line in the subsequent function might also need this done to it unless it ever gets called with timestamp format and arguments. Let me know if you'd like for me to send the pull request, I'd be glad to help.
